### PR TITLE
dependencies and tidying

### PR DESCRIPTION
**Pull request by alan-y**
Would be great if you could give this a quick review please? Not too many changes.

### CODE REVIEW

**Comment by RoryMadigan** [comment]
Hi Alan, I ran the following line when trying to install your branch from a downloaded file, but received the error…
R> devtools::install_local("C:/Users/roryma01/Downloads/phiproject-alan.zip")
Error: Failed to install 'phiproject-alan.zip' from local:
  Timeout was reached: [api.github.com] Connection timed out after 10014 milliseconds

Do you know what might have happened there?
Is there some sort of GitHub connection setting that I have to adjust?
Thanks,

Rory

**Comment by alan-y** [comment]
> R> devtools::install_local("C:/Users/roryma01/Downloads/phiproject-alan.zip") Error: Failed to install 'phiproject-alan.zip' from local: Timeout was reached: [api.github.com] Connection timed out after 10014 milliseconds

Ah interesting thanks for this feedback Rory - it's the dependency on phsmethods and it's obviously trying to download it by default with `install_local()`, you will need to use `install_local("C:/Users/roryma01/Downloads/phiproject-alan.zip", dependencies = FALSE)`

**Comment by jackhannah95** [comment]
I've got a bit of a backlog of PRs to review at the moment and might struggle to get to this one soon. If you're happy with it @RoryMadigan then feel free to merge.

File: DESCRIPTION [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Package: phiproject
-Title: PHI R Project Template
+Title: PHI R Templates
 Authors@R: c(
     person("Anna", "Price", email = "anna.price5@nhs.net", role = c("aut", "cre")),
     person("Alan", "Yeung", email = "alan.yeung@nhs.net", role = "aut", comment = c(ORCID = "0000-0001-5226-3695"))
```
Comment (jackhannah95) [inline DESCRIPTION]: Surely you can take on the job of maintainer now @alan-y? 👀

**Comment by RoryMadigan** [comment]
No problem Alan,
Now I tried that and then it said that I needed phimethods, which I then tried
installing with …
install_local("C:/Users/roryma01/Downloads/phsmethods-master.zip")

It looked like it was working, but then an error came up saying …
Error: Failed to install 'phsmethods-master.zip' from local:
  (converted from warning) cannot remove prior installation of package ‘glue’

I then tried uninstalling the package ‘glue’ which looked like it uninstalled, but still
ended up with the same error.  Do you have any idea what be going on here?
Thanks,

Rory

**Comment by RoryMadigan** [comment]
Hi Jack,
That’s no problem, I can handle this one.

Rory

**Comment by jackhannah95** [comment]
> It may be better if I shift phsmethods into `Suggests` as it's not currently using anything from it directly - what do you think @jackhannah95 ?

Yeah I'd say get rid of it for now if it's not needed.

File: DESCRIPTION (hunk 1 above)
Comment (alan-y) [reply DESCRIPTION]: haha I suppose I could update this but I wanted to make sure Anna is credited with her contribution first

**Comment by alan-y** [comment]
> No problem Alan, Now I tried that and then it said that I needed phimethods, which I then tried installing with … install_local("C:/Users/roryma01/Downloads/phsmethods-master.zip") It looked like it was working, but then an error came up saying … Error: Failed to install 'phsmethods-master.zip' from local: (converted from warning) cannot remove prior installation of package ‘glue’ I then tried uninstalling the package ‘glue’ which looked like it uninstalled, but still ended up with the same error. Do you have any idea what be going on here? Thanks, Rory

I'll move phimethods to `Suggests` and it should sort this problem. And it'll hopefully fix this problem.

**Pull request merged**